### PR TITLE
Implemented missing TMs/HMs type palettes

### PR DESCRIPTION
**Pull request by LOuroboros**
Addresses #318

### CODE REVIEW

**Comment by DizzyEggg** [comment]
I think you forgot about actual images Lunos

**Comment by LOuroboros** [comment]
A showcase? Sure, here you go:


If I were to assign them to a TM or HM, I would just use the exact same sprite the rest of the TMs and HMs use in `src/data/item_icon_table.h`. I didn't assign them to any because the item_expansion uses the same set of TMs and HMs that Pokémon Emerald. It doesn't have any data for new TMs or HMs of either Bug or Fairy type.

File: src/data/graphics/items.h [hunk 1]
```diff
@@ -1514,3 +1514,7 @@ const u32 gItemIconPalette_SteelMemory[] = INCBIN_U32("graphics/items/icon_palet
 
 const u32 gItemIcon_WaterMemory[] = INCBIN_U32("graphics/items/icons/water_memory.4bpp.lz");
 const u32 gItemIconPalette_WaterMemory[] = INCBIN_U32("graphics/items/icon_palettes/water_memory.gbapal.lz");
+
+const u32 gItemIconPalette_BugTMHM[] = INCBIN_U32("graphics/items/icon_palettes/bug_tm_hm.gbapal.lz");
+
```
Comment (DizzyEggg) [inline src/data/graphics/items.h:1519]: could you get rid of that new line there?

Comment (LOuroboros) [reply src/data/graphics/items.h:1519]: I added it for consistency's sake. If you look at how the paths to the other TM/HM sprite palettes are defined above in the file, they're also separated from each other by an empty line.

**Pull request merged**